readthedocs/proxito/views/hosting.py

```diff
@@ -452,6 +454,9 @@ def _v1(self, project, version, build, filename, url, request):
                 "analytics": {
                     "code": settings.GLOBAL_ANALYTICS_CODE,
                 },
+                "resolver": {
+                    "filename": escape(iri_to_uri(filename)),
```

Comment:
There is no need to escape or encode the field here as this isn't being injected directly into the page as is with the header version.